matrix/common/constants.py

```diff
@@ -52,6 +52,8 @@ class BundleType(Enum):
             analysiskey        VARCHAR(60) NOT NULL,
             barcode            VARCHAR(32),
             genes_detected     INTEGER,
+            total_umis         INTEGER,
+            emptydrops_is_cell BOOLEAN,
```

Comment:
Can these be more explicit about being tied to Optimus? I considered a comment here (I usually use this file as a quick reference to the current table schema), or even better if the column name could reflect Optimus?

Reply:
Yeah, actually would be interested in a suggestion from the pipelines team on this one. I think these fields should apply to all droplet scRNA-seq though.